apps/src/code-studio/soundLibrary.json

```diff
@@ -3724,4 +3724,4 @@
       "category_male_voiceover/you_win_male"
     ]
   }
-}
\ No newline at end of file
+}
```

Comment:
It'd be nice to get `soundLibrary.json` entirely out of this PR, but understandable if that's tricky at this point.